go/host/enclave/state.go

```diff
@@ -126,12 +126,7 @@ func (s *StateTracker) OnEnclaveStatus(es common.Status) {
 	s.m.Lock()
 	defer s.m.Unlock()
 	s.enclaveStatusCode = es.StatusCode
-	if es.L1Head != gethutil.EmptyHash {
```

Comment:
We don't need this hack any more, if enclave has no head we don't try to pretend its head is the configured head, we just check for that and use configured start hash.